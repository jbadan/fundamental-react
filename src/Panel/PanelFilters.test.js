import { mount } from 'enzyme';
import Panel from './Panel';
import React from 'react';
import renderer from 'react-test-renderer';

describe('<PanelFilters />', () => {
    const panelFilters = (
        <Panel.Filters>
            <div>Panel Filters</div>
            <br />
        </Panel.Filters>
    );

    test('create panel filters', () => {
        let component = renderer.create(panelFilters);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    describe('Prop spreading', () => {
        test('should allow props to be spread to the PanelFilters component', () => {
            const element = mount(<Panel.Filters data-sample='Sample' />);

            expect(
                element.getDOMNode().attributes['data-sample'].value
            ).toBe('Sample');
        });
    });
});
