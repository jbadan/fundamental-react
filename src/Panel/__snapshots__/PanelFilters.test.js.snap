// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PanelFilters /> create panel filters 1`] = `
<div
  className="fd-panel__filters"
>
  <div>
    Panel Filters
  </div>
  <br />
</div>
`;
